.circleci/config.yml

```diff
@@ -0,0 +1,101 @@
+version: 2
+
+.base: &base
+  docker:
+      - image: trojsten/web-base:latest
+      - image: postgres:9.5
+        environment:
+          POSTGRES_USER: trojsten
+          POSTGRES_DB: trojsten
+      # - image: elasticsearch:2.4.6
+  working_directory: ~/web
+
+
+jobs:
+  checkout_code:
+    <<: *base
+    steps:
+      - checkout
+      - save_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+          paths:
+            - ~/web
+  build_dependencies:
+    <<: *base
+    steps:
+      - restore_cache:
+          key: v1-repo-{{ .Environment.CIRCLE_SHA1 }}
+      - restore_cache:
+          keys:
+            - v2-dependencies-{{ checksum "requirements.txt" }}
+            # fallback to using the latest cache if no exact match is found
+            - v2-dependencies-
```

Comment:
This looks like this is unused, because I don't see a `save_cache` with a key `v2-dependencies-` without the requirements.txt checksum anywhere.., don't we need to save the cache to 2 different keys (one with checksum and one without) for this to work as expected by the above comment?

Reply:
I took this from the sample config, and I think it uses prefix (so if the checksum does not match it finds it anyway). I think this behaviour is ok with requirements.txt because we pin the versions, so we should always install whatever is in the requirements regardless the cache. The cache just speeds up the process because we don install what is already installed.

Having the fallback key prefix is useful to speed up the build when we change the requirements.

Reply:
oh, that's actually a cool feature i didn't know about 👍